websites/serializers.py

```diff
@@ -34,4 +34,32 @@ class WebsiteSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Website
-        fields = "__all__"
+        fields = [
+            "uuid",
+            "created_on",
+            "updated_on",
+            "name",
+            "title",
+            "source",
+            "publish_date",
+            "metadata",
+            "starter",
+        ]
+
+
+class WebsiteDetailSerializer(serializers.ModelSerializer):
+    """ Serializer for websites with serialized config """
+
+    config = serializers.SerializerMethodField()
```

Comment:
Is this just for convenience? I guess having `config` as a top-level property is nice to have but it seems more straightforward just to use the detail serializer for `starter` (`starter = WebsiteStarterDetailSerializer(read_only=True)`)

Reply:
That's true, I'll do that